config/application.rb

```diff
@@ -1,6 +1,19 @@
 require_relative 'boot'
 
+require "rails"
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+# require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
+require "action_view/railtie"
+# require "action_cable/engine"
+# require "sprockets/railtie"
+require "rails/test_unit/railtie"
```

Comment:
did it crash without the uncommented `requires`? I see you added some commented, but some uncommented... weird that it only had `action_controller/railtie` before...